docs/UserGuide.md

```diff
@@ -464,7 +480,9 @@ To illustrate, suppose you have 5 buyers in total (Adam, Ben, Carl, Daniel, Elle
 </div>
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-You can use the `list` command to reset the display lists to display all the available buyers before using `match property`. This will find compatible buyers from the full pool of available buyers.
+
+You can use the [`list`](#listing-all-propertiesbuyers--list) command to reset the display lists to display all the available buyers before using `match property`. This will allow you to find compatible buyers from the full pool of available buyers so that you don't miss out on the *best* matches.
```

Comment:
Can add the link`(#listing-all-propertiesbuyers--list)` in the `find` command tip too? Thks!